src/test/ui/rfc-2294-if-let-guard/feature-gate.stderr

```diff
@@ -17,7 +17,7 @@ LL |         () if let 0 = 1 => {}
    = help: add `#![feature(if_let_guard)]` to the crate attributes to enable
 
 error[E0658]: `if let` guard is not implemented
```

Comment:
This should probably say "`if let` guard is experimental".

Reply:
I'll file a PR fixing this later today